lib/Alembic/AbcCoreOgawa/StreamManager.cpp

```diff
@@ -67,6 +67,14 @@ Alembic::Util::int64_t ffsll( Alembic::Util::int64_t iValue )
     return 0;
 }
 
+#elif defined( __HAIKU__ )
+
+#define COMPARE_EXCHANGE( V, COMP, EXCH ) __atomic_compare_exchange_n( &V, &COMP, EXCH, false, __ATOMIC_SEQ_CST, __ATOMIC_SEQ_CST )
+#include <strings.h>
+int ffsll(long long i)
+{
+	return (__builtin_ffsll(i));
+}
```

Comment:
If I move this to the location as mentioned by @extrowork in https://github.com/alembic/alembic/pull/230/files#diff-92d170a966b915b03c807af16cca1f8efbd629060ea1be76fd7f739a64aecd0cR113 it works ok, like this I still get ffsll not declared on Haiku

Reply:
Does it just need to be tweaked to return an int64_t?  (and to take an int64_t?)

Reply:
Pinging @extrowork here, I'm not a developer so not really in a position to answer this :) I just came across this while trying to bring our port to the latest version atm